example/lib/place_source.dart

```diff
@@ -134,13 +134,28 @@ class PlaceSymbolBodyState extends State<PlaceSymbolBody> {
                       child: const Text('Remove source (asset image)'),
                       onPressed: sourceAdded
                           ? () async {
-                              await removeLayer(LAYER_ID);
```

Comment:
why did you remve the await here?

Reply:
I notice that when you await on `removeLayer` it won't run the code below. (on iOS)

Reply:
this is fixed on master now - please rebase and put the await back